net/nimble/host/src/ble_gap.c

```diff
@@ -2154,6 +2155,66 @@ ble_gap_disc_tx_params(uint8_t own_addr_type,
     return 0;
 }
 
+#if MYNEWT_VAL(BLE_EXT_ADV)
+static int
+ble_gap_ext_disc_tx_params(uint8_t own_addr_type, uint8_t filter_policy,
+                       const struct ble_hs_hci_ext_scan_param *uncoded_params,
+                       const struct ble_hs_hci_ext_scan_param *coded_params)
+{
+    uint8_t buf[BLE_HCI_CMD_HDR_LEN + BLE_HCI_LE_EXT_SCAN_BASE_LEN +
+                                2 * BLE_HCI_LE_EXT_SCAN_SINGLE_PARAM_LEN];
+    uint8_t phy_mask = 0;
+    struct ble_hs_hci_ext_scan_param param[2] = {0};
```

Comment:
I'm sorry to cause so much trouble with that nitpick, but I don't think this will compile.  Since it's an array, you need two initializers, e.g., `{{0}}`.